File: python/svix/api_internal/message.py
# This file is @generated
from .common import ApiBaseAsync, ApiBaseSync
from .message_pollerv2 import (
    MessagePollerv2,
    MessagePollerv2Async,
)


class MessageAsync(ApiBaseAsync):
    @property
    def pollerv2(self) -> MessagePollerv2Async:
        return MessagePollerv2Async(self._client, self._httpx_client)


class Message(ApiBaseSync):
    @property
    def pollerv2(self) -> MessagePollerv2:
        return MessagePollerv2(self._client, self._httpx_client)

File: php/src/ApiInternal/Message.php
<?php

// this file is @generated
declare(strict_types=1);

namespace Svix\ApiInternal;

use Svix\Request\SvixHttpClient;

class Message
{
    public MessagePollerv2 $pollerv2;

    public function __construct(
        private readonly SvixHttpClient $client,
    ) {
        $this->pollerv2 = new MessagePollerv2($client);
    }
}
